### DISCUSSION TITLE
Firestore not in synch

### DISCUSSION THREAD

**Discussion by ItaliaNorway** [AntlerVC/firetable#1480] [Support / Q&A]
I have a table in Rowy, I deleted some columns but when I query firestore I still see those data. Why? 
When I create a new row I see it in firestore so is synching

**Comment by bofeiw** [comment]
Deleting the column in Rowy does not delete the data in your Firestore. It only deletes the column info metadata of the Rowy table.

**Reply by ItaliaNorway** [reply]
so to delete a colum I need to do it manually in firestore?

**Reply by Chadha93** [reply]
Yes, you can try deleting a column.

Also, Firestore has a built-in caching mechanism that helps improve read performance. It is possible that the data you see is being fetched from the cache instead of the actual database. To resolve this, you can try clearing the cache or disabling it temporarily for testing purposes. You can clear the cache by restarting your application or by using the clearPersistence() method in Firestore

**Reply by ItaliaNorway** [reply]
I deleted a column in firestore and added few others... nothing changes in Rowy.
Should not sync constantly? 
The same when I added a column in Rowy I don´t see it in firestore... 
When I change or add a Row the sync is almost in real time. 
How Rowy works?

**Reply by bofeiw** [reply]
@ItaliaNorway Hey sorry for the late response. 

Rowy syncs data in real time with Firestore. When you make changes to row data in Rowy, you should see it updated in Firestore and vice versa.

Columns, are metadata of the table stored in the `_rowy_` collection. When you add a column, it updates the metadata, and the Firestore table data remains same. Firestore table data only updates when you make updates to the row data **after** you add the column.
